.github/workflows/linux.yml

```diff
@@ -233,7 +233,7 @@ jobs:
   #################################################################################
   gcc9:
     # Purpose is to examine builds with oldest-supported gcc and test with pydata/sparse.
-    name: Oldest GCC & pydata/sparse, fast, py3.11/npMin, pip+pytest
+    name: Oldest GCC & pydata/sparse, full, py3.11/npMin, pip+pytest
```

Comment:
This was just a mistake in the job name I caught. Running `pytest --pyargs scipy` will run the full rather than the fast test suite.